lib/book_parser.py

```diff
@@ -191,7 +181,7 @@ def find_erl_and_encoding(self):
             nrecs, = unpack('>H', book_data[76:78])
             recs_start = 78 + (nrecs * 8) + 2
             erl, = unpack('>L', book_data[recs_start + 4:recs_start + 8])
-            codec = 'latin-1' if unpack('>L', book_data[recs_start + 28:recs_start + 32])[0] == 1252 else 'utf8'
+            codec = 'cp1252' if unpack('>L', book_data[recs_start + 28:recs_start + 32])[0] == 1252 else 'utf8'
```

Comment:
just realized that i was using the wrong thing before according to some docs on mobi headers